server/controllers/refer/peopleController.test.ts

```diff
@@ -0,0 +1,68 @@
+import type { DeepMocked } from '@golevelup/ts-jest'
+import { createMock } from '@golevelup/ts-jest'
+import type { NextFunction, Request, Response } from 'express'
+import createError from 'http-errors'
+
+import PeopleController from './peopleController'
+import type { Person } from '../../@types/models/Person'
+import type PersonService from '../../services/personService'
+import { personFactory } from '../../testutils/factories'
+import personUtils from '../../utils/personUtils'
+
+describe('PeopleController', () => {
+  const token = 'SOME_TOKEN'
+  const request: DeepMocked<Request> = createMock<Request>({ user: { token } })
+  const response: DeepMocked<Response> = createMock<Response>({})
+  const next: DeepMocked<NextFunction> = createMock<NextFunction>({})
+
+  const personService = createMock<PersonService>({})
+
+  let peopleController: PeopleController
+
+  beforeEach(() => {
+    peopleController = new PeopleController(personService)
+  })
+
+  describe('show', () => {
+    let person: Person
+
+    beforeEach(() => {
+      person = personFactory.build({
+        name: 'Del Hatton',
+      })
+      personService.getPerson.mockResolvedValue(person)
+    })
+
+    it('renders the confirmPerson template with the found Person', async () => {
+      personService.getPerson.mockResolvedValue(person)
+
+      const requestHandler = peopleController.show()
+
+      await requestHandler(request, response, next)
+
+      expect(response.render).toHaveBeenCalledWith('referrals/confirmPerson', {
+        pageHeading: "Confirm Del Hatton's details",
+        personSummaryListRows: personUtils.summaryListRows(person),
+      })
+    })
+
+    it('throws an error if the person is not found', async () => {
+      const notFoundRequest: DeepMocked<Request> = createMock<Request>({
+        user: { token },
+        params: {
+          prisonNumber: 'NOTFOUND',
+        },
+      })
```

Comment:
Is this needed? Isn't the mocked service method the main thing here?

Reply:
Ah I was using this to test the error message, but if the error message isn't being picked up, then this is unnecessary, although it'd be good to figure out how to get it to test the error message!

Reply:
Is the error message definitely working as expected?

Reply:
It's outputted in the logs, but not to the user (which I think is what we want)

Reply:
As above, I've tweaked this to not make an assertion on the user message for now.